web-app/react-app/components/AuthPage/shared/Button/button.pcss

```diff
@@ -0,0 +1,24 @@
+@import "@streamr/streamr-layout/pcss/variables.pcss";
+
+.button {
+  background: #0324FF;
+  border: 0;
+  border-radius: 2px;
+  color: white;
+  cursor: pointer;
+  flex-shrink: 0;
+  font-family: 'IBM Plex Mono', monospace;
+  font-size: 0.875em;
+  font-weight: 600;
+  line-height: 3em;
+  outline: 0 !important;
+  padding: 0 2em;
```

Comment:
I think this should be `padding: 0 2.5em;` to match a 90px 'GO/NEXT' desktop buttons
https://share.goabstract.com/25ec7441-22f4-4886-8f49-6cce33dc7c0f

Reply:
[CORE-1497](https://streamr.atlassian.net/browse/CORE-1497)